subsys/bluetooth/shell/bt.c

```diff
@@ -697,6 +697,35 @@ static int cmd_name(const struct shell *sh, size_t argc, char *argv[])
 	return 0;
 }
 
+static int cmd_appearance(const struct shell *sh, size_t argc, char *argv[])
+{
+	if (argc == 1) {
+		shell_print(sh, "Bluetooth Appearance: 0x%04x", bt_get_appearance());
+		return 0;
+	}
+
+	if (argc == 2) {
+		int err;
+		uint16_t new_appearance;
+
+		if (strlen(argv[1]) != 6 || strncmp(argv[1], "0x", 2) || !hex2bin(
+				&argv[1][2], strlen(&argv[1][2]), ((uint8_t *)&new_appearance),
```

Comment:
And split the `hex2bin()` call onto its own line, or into a separate if-statement. You generally also seem to have trouble aligning the split lines - they should start right after the appropriate `(` on the preceding line.

Reply:
Thanks. Can we get check_compliance.py to enforce this?

Reply:
Is it fine like it's now after changing the indent, or did you mean this?:
```c
	val = argv[1];

	if (strlen(val) != 6) {
		shell_error(sh, "Argument must be 0x followed by exactly 4 hex digits.");
		return -EINVAL;
	}

	if (strncmp(val, "0x", 2)) {
		shell_error(sh, "Argument must be 0x followed by exactly 4 hex digits.");
		return -EINVAL;
	}

	if (!hex2bin(&val[2], strlen(&val[2]), ((uint8_t *)&app), sizeof(app)))
		shell_error(sh, "Argument must be 0x followed by exactly 4 hex digits.");
		return -EINVAL;
	};

	app = sys_be16_to_cpu(app);
```